musicbrainzngs/mbxml.py

```diff
@@ -293,6 +295,19 @@ def parse_event(event):
 
     return result
 
+def parse_instrument(instrument):
+    result = {}
+    attribs = ["id", "type", "ext:score"]
+    elements = ["name", "description"]
```

Comment:
According to https://github.com/metabrainz/mmd-schema/blob/master/schema/musicbrainz_mmd-2.0.rng#L912 instruments can have disambiguation comments and https://beta.musicbrainz.org/instrument/dabdeb41-560f-4d84-aa6a-cf22349326fe does indeed have one.